deployment/postgres-init-db/sql/tables/sequencing/expanded_sequences.sql

```diff
@@ -5,16 +5,17 @@ create table sequencing.expanded_sequences (
   seq_id text not null,
   simulation_dataset_id int not null,
   expanded_sequence jsonb not null,
-  edsl_string text not null,
```

Comment:
It will be dropped. We will not be using this column moving forward. Sequences that have been expanded will only store the seqJSON. We will be converting the seqJSON to SeqN.txt in the editor.

Reply:
Regarding the `down.sql`, is this data recoverable? If not, we should call it out as one of the breaking changes.

Reply:
Sadly the data will not be recoverable. Is this something we let @dandelany know about? I don't really know the formal way of reporting this.